migrate/migration.go

```diff
@@ -25,11 +25,11 @@ type Migration struct {
 	Down MigrationFunc `bun:"-"`
 }
 
-func (m *Migration) String() string {
+func (m Migration) String() string {
 	return m.Name
 }
 
-func (m *Migration) IsApplied() bool {
+func (m Migration) IsApplied() bool {
```

Comment:
This now copies the whole `Migration` struct which should be kinda slowish.  I am going to merge this PR anyway because the performance here is not crucial.